srcalc/src/main/java/gov/va/med/srcalc/domain/model/DiscreteNumericalVariable.java

```diff
@@ -222,6 +222,36 @@ final void setUpperBound(final float upperBound)
         {
             fUpperBound = upperBound;
         }
+        
+        /**
+         * Returns the upper bound as a String.
```

Comment:
You should document why this and the next method are package-private, and include why this was necessary at all.